### ISSUE TITLE
Color in preview 

### ISSUE THREAD

**Issue by ant31** [heptio/vscode-jsonnet#2]
First, this extension is awesome, I love it, made me move to vscode :) 

Is the preview window can reuse the syntax color from json ? Isn't only a mime-type to set ?

**Comment by hausdorff** [comment]
@ant31 Yeah, good question. This extension is pretty bare-bones for now, but it has been on my list to add this for some time. I'm not sure when I'll get around to it -- hopefully soon.
